Middleware/src/test/data/Mock energy consumption data - Energir.csv

```diff
@@ -0,0 +1,202 @@
+RTA,SCIAN,Nombre de client HQ,Nombre de clients É,Janvier,Février,Mars,Avril,Mai,Juin,Juillet,Août,Septembre,Octobre,Novembre,Décembre,

```

Comment:
Please add anaonymized CSV files to this folder, remove any files not being used.